src/CalcViewModel/Common/NavCategory.cpp

```diff
@@ -49,151 +44,260 @@ static constexpr int CURRENCY_ID = 16;
 static constexpr int GRAPHING_ID = 17;
 // ^^^ THESE CONSTANTS SHOULD NEVER CHANGE ^^^
 
+wchar_t* towchar_t(int number)
+{
+    auto wstr = to_wstring(number);
+    return _wcsdup(wstr.c_str());
+}
+
+extern "C"
+{
+    WINADVAPI LSTATUS APIENTRY RegGetValueW(
+        _In_ HKEY hkey,
+        _In_opt_ LPCWSTR lpSubKey,
+        _In_opt_ LPCWSTR lpValue,
+        _In_ DWORD dwFlags,
+        _Out_opt_ LPDWORD pdwType,
+        _When_(
+            (dwFlags & 0x7F) == RRF_RT_REG_SZ || (dwFlags & 0x7F) == RRF_RT_REG_EXPAND_SZ || (dwFlags & 0x7F) == (RRF_RT_REG_SZ | RRF_RT_REG_EXPAND_SZ)
+                || *pdwType == REG_SZ || *pdwType == REG_EXPAND_SZ,
+            _Post_z_) _When_((dwFlags & 0x7F) == RRF_RT_REG_MULTI_SZ || *pdwType == REG_MULTI_SZ, _Post_ _NullNull_terminated_)
+            _Out_writes_bytes_to_opt_(*pcbData, *pcbData) PVOID pvData,
+        _Inout_opt_ LPDWORD pcbData);
+}
+
+bool IsGraphingModeAvailable()
+{
+    static bool supportGraph = Windows::Foundation::Metadata::ApiInformation::IsMethodPresent("Windows.UI.Text.RichEditTextDocument", "GetMath");
+    return supportGraph;
+}
+
+Box<bool> ^ _isGraphingModeEnabledCached = nullptr;
+bool IsGraphingModeEnabled()
+{
+    if (!IsGraphingModeAvailable())
+    {
+        return false;
+    }
+
+    if (_isGraphingModeEnabledCached != nullptr)
+    {
+        return _isGraphingModeEnabledCached->Value;
+    }
+    DWORD allowGraphingCalculator{ 0 };
+    DWORD bufferSize{ sizeof(allowGraphingCalculator) };
+    // Make sure to call RegOpenKey only on Windows 10 1903+
+    if (!RegGetValueW(
+            HKEY_LOCAL_MACHINE,
+            L"SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Policies\\Calculator",
+            L"AllowGraphingCalculator",
+            RRF_RT_REG_DWORD,
+            nullptr,
+            reinterpret_cast<LPBYTE>(&allowGraphingCalculator),
+            &bufferSize) != ERROR_SUCCESS)
+    {
+        _isGraphingModeEnabledCached = true;
+        return true;
+    }
+
+    _isGraphingModeEnabledCached = allowGraphingCalculator != 0;
```

Comment:
>allowGraphingCalculator [](start = 35, length = 23)

I think there's a bug here...  I do not have the reg key set but allowGraphingCalculator is set to 0 which is graying out the option.

Reply:
Fixed!